### PR TITLE
Feature/show hide password

### PR DESCRIPTION
**Pull request by neighbor983**
Added a visibility button to the password fields. Added the angular material packages. 

P.S. Change the UI to angular material will be a separate story.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,13 +16,15 @@
     "postinstall": "ng build"
   },
   "dependencies": {
+    "@angular/animations": "^4.0.2",
```
Comment (dghalbr) [inline package.json]: Should this be the same version as the rest of our angular dependencies?

**Comment by dghalbr** [comment]
I made a comment in the package.json. Also, this should be on the login page too. Lastly, it needs some css I think. Give it a cursor of pointer so that when you mouse over it then it looks clickable. and make it center with the input. What did you use hammerjs for? I didn't see any implementation. I may just be blind...

File: package.json (hunk 1 above)
Comment (neighbor983) [inline package.json]: I should just remove the angular material parts.

File: src/app/common/show-hide-password-container/show-hide-password-container.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  show = false;
+ 
+   @ContentChild('showhideinput') input;
+ 
+   constructor(){}
+ 
+   toggleShow()
+   {
+      this.show = !this.show;
+      console.log(this.input);
```
Comment (dghalbr) [inline src/app/common/show-hide-password-container/show-hide-password-container.component.ts]: probably shouldn't console log password stuffs? maybe?

**Pull request merged**